content/actions/reference/events-that-trigger-workflows.md

```diff
@@ -606,7 +606,7 @@ on:
 
 {% note %}
 
-**Note:** The webhook payload available to GitHub Actions does not include the `added`, `removed`, and `modified` attributes in the `commit` object. You can retrieve the full commit object using the REST API. For more information, see "[Get a single commit](/rest/reference/repos#get-a-single-commit)"".
+**Note:** The webhook payload available to GitHub Actions does not include the `added`, `removed`, and `modified` attributes in the `commit` object. You can retrieve the full commit object using the REST API. For more information, see "[Get a commit](/rest/reference/repos#get-a-commit)"".
```

Comment:
```suggestion
**Note:** The webhook payload available to GitHub Actions does not include the `added`, `removed`, and `modified` attributes in the `commit` object. You can retrieve the full commit object using the REST API. For more information, see "[Get a commit](/rest/reference/repos#get-a-commit)".
```
Thanks so much for opening an issue and a PR for this fix @0xdevalias! Your update looks great ✨ ! I noticed while reviewing it that there's an extra `"` right after the link, so I removed it in the suggestion above. Let me know once you accept that and I'll get this merged in 💖

Reply:
Good catch. Change accepted :)